### PR TITLE
Upgrade to 2.17

### PR DESCRIPTION
**Pull request by toddjordan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,32 +12,33 @@
   "repository": "",
   "scripts": {
     "build": "ember build",
-    "start": "ember server",
+    "start": "ember serve",
```
Comment (locks) [inline package.json:15]: when did this change?

**Review by locks — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (jenweber) [reply package.json:15]: Looks like we don't have 1 consistent command. CLI docs say server, and various parts of the guides say serve

**Review by jenweber — APPROVED** [review]
Ran this locally and inspected manual ember init diffs. Looks good.

**Comment by locks** [comment]
We haz conflicts in package.json.

File: package.json (hunk 1 above)
Comment (toddjordan) [reply package.json:15]: it changed bc the default blueprint changed

**Comment by toddjordan** [comment]
conflicts resolved

**Comment by acorncom** [comment]
LGTM

**Pull request merged**